src/config/config_type.rs

```diff
@@ -571,7 +571,11 @@ macro_rules! create_config {
 
                 eprintln!(
                     "Warning: the `version` option is deprecated. \
-                    Use `style_edition` instead."
+                    Use `style_edition=\"{0}\"` instead.",
+                    match self.version.2 {
+                        Version::One => "2015",
+                        Version::Two => "2024",
+                    }
```

Comment:
`version=One` is equivalent to `2015`,  `2018`, or `2021`. Do you think it would be useful to be a little more specific in the warning message?

Reply:
If I understand correctly there is no difference between using `2015`, `2018` or `2021` so I think it should be fine to give the "default" value as an hint in the warning, given that following it will give you the desired behavior. It could maybe useful to mention it in the docs so to not have to look it up in the tracking issue.

If we do that should the `style_version=2027` also be added as an alternative for `2024`?

Reply:
Mentioning it in the docs sounds good to me, and we can leave the warning message recommending the default value. We definitely don't want to recommend `style_version=2027` as an alternative for 2024 since 2024 will be stabilized soon and 2027 is going to remain an unstable variant for now.